cfi_forward.adoc

```diff
@@ -377,5 +377,6 @@ current `ELP` and `ELP` is set to `NO_LP_EXPECTED`.
 
 `MRET` or `SRET` instruction is used to return from a trap in M-mode or S-mode
 respectively. When executing an `xRET` instruction, the `ELP` is set to `xPELP`
-and xPELP is set to `NO_LP_EXPECTED`. The trap handler should restore the
-preserved `lpl` value before invoking `SRET` or `MRET`.
+if `xFCFIE` is 1 at privilege level. The `xPELP` is set to `NO_LP_EXPECTED`.
```

Comment:
Maybe say at the targeted privilege level? It seems a little vague as is.

Reply:
agree